entities/general/content-license.md

```diff
@@ -0,0 +1,27 @@
+Content License - Entities
+==========================
+
+Table of Content
+----------------
+<!-- TOC START min:1 max:3 link:true asterisk:false update:true -->
+  - [Class and Schema](#class-and-schema)
+  - [Entities](#entities)
+<!-- TOC END -->
+
+## Class and Schema
+
+- Class Name: ["Class Name"](../../classes/general/content-license.md)
+- Class Id : `3`
+- Supported schema Id(s): `0`
+
+## Entities
+
+'Permission info'. 'Property info and link to [property type(s)](../../README.md#property-types)'.
+The table below lists all available entities:
+
+| Entity ID | Value                                     | Attribution | Further Information                                                           |
+|-----------|-------------------------------------------|-------------|-------------------------------------------------------------------------------|
+| 191       | "Public Domain"                           | Not needed  | [link](https://en.wikipedia.org/wiki/Public_domain)                           |
+| 192       | "Original Content"                        | Not needed  | [Terms of Service](https://testnet.joystream.org/pioneer/#/pages/tos)         |
+| 193       | "Creative Commons (attribution required)" | Required    | [link](https://creativecommons.org/licenses/)                                 |
+| 194       | "Fair Use (attribution required"          | Required    | [link](https://cmsimpact.org/code/code-best-practices-fair-use-online-video/) |
```

Comment:
Not really a license, perhaps best to remove for now.

Reply:
(Fair Use)

Reply:
Same:
Should maybe have been rephraseg, but this is already live...